prime-angular-frontend/src/app/modules/enrolment/pages/access-agreement/access-agreement.component.ts

```diff
@@ -86,7 +106,7 @@ export class AccessAgreementComponent extends BaseEnrolmentPage implements OnIni
             (result)
               ? this.enrolmentResource.submissionAction(
                 this.enrolment.id,
-                isAcceptingToa ? SubmissionAction.ACCEPT_TOA : SubmissionAction.DECLINE_TOA)
+                isAcceptingToa ? SubmissionAction.ACCEPT_TOA : SubmissionAction.DECLINE_TOA, this.accessAgreementGuid.value)
```

Comment:
For ease of reading maybe put the 3rd param on a newline with the closing ) also on a newline, or wrap the exhaust map in { } and move ternary param up into a locally scoped const and one liner the method invocation